src/plugins/news/screens/article/delegate.js

```diff
@@ -9,6 +13,31 @@ class Delegate extends AbstractDelegate {
       formName: formNames.ARTICLE,
     }, dependencies);
   }
+
+  handleSubmit(data, formDispatch, formProps) { // eslint-disable-line consistent-return
+    const { blocksmithState, dispatch } = this.component.props;
+    const { isValid, validEditorState } = validateBlocks(blocksmithState.editorState);
+
+    if (isValid) {
+      return AbstractDelegate.prototype.handleSubmit.call(this, data, formDispatch, formProps);
```

Comment:
How about calling the parent method like this?

```suggestion
      return super.handleSubmit(data, formDispatch, formProps);
```

Reply:
@arunkarnati good call, updated